frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -25,7 +25,7 @@ export let DownloadAnalysisLink = withStore()(({ store }) =>
     icon={<DownloadIcon />}
     label='Download'
     labelPosition='before'
-    onClick={onClick(store)}
+    onClick={() => onClick(store)}
```

Comment:
what does this do?

Reply:
Previously, the value of the onClick attribute of the button was being set to what was returned by the onClick function defined in this file (which was another function). Now the value of the onClick attribute of the button _is_ the onClick function defined in this file.